### DISCUSSION TITLE
Does MotionDirector work on Stable Video Diffusion?

### DISCUSSION THREAD

**Discussion by TruthSearcher** [showlab/MotionDirector#25] [Q&A] [answered]

**Comment by ruizhaocv** [comment]
This matter is on our to-do list, but we currently don't have a specific estimated time to finish it. 
We welcome everyone to try adapting MotionDirector to SVD and collaborate in enhancing this project together. Thanks!

**Comment by ruizhaocv** [comment]
Thank you. I think it would be better to keep this issue open for others who have similar questions.

**Comment by dailingx** [comment]
I'm also interested in this

**Comment by olliacc** [comment]
We would also like to know if MotionDirector works motion modules like animatediff? @ruizhaocv

**Comment by ruizhaocv** [comment]
Hi olliacc. Thanks for your question. The answer is the same with the SVD. It is on our to-do list, but we don't have an estimated time to finish it. We welcome everyone to adapt MotionDirector to variant base models and collaborate in enhancing this project together.

**Comment by howardgriffin** [comment]
Any update?

**Comment by ruizhaocv** [comment, marked as answer]
Hi all. The MotionDirector training for AnimateDiff is available at https://github.com/ExponentialML/AnimateDiff-MotionDirector. Thank @ExponentialML very much for this awesome repo.

**Comment by eugenelyj** [comment]
@ruizhaocv Dear author, I am trying to implement the SVD version but I have two questions and maybe you have some ideas?
1. How to modify the spatial loss to adapt to the image-to-video pipeline of SVD?
2. How to modify the temporal loss to adapt to the v-prediction formula of SVD?
